--- antlr4-4.9.3/runtime/Cpp/CMakeLists.txt	2022-12-20 18:47:31.098294558 +0000
+++ antlr4-4.9.3/runtime/Cpp/CMakeLists.txt	2022-12-20 18:48:49.216083290 +0000
@@ -39,10 +39,6 @@
   CMAKE_POLICY(SET CMP0054 OLD)
 endif()
 
-if(CMAKE_SYSTEM_NAME MATCHES "Linux")
-  find_package(PkgConfig REQUIRED)
-  pkg_check_modules(UUID REQUIRED uuid)
-endif()
 if(APPLE)
   find_library(COREFOUNDATION_LIBRARY CoreFoundation)
 endif()
